modules/P02_firmware_bin_file_check.sh

```diff
@@ -34,7 +35,9 @@ P02_firmware_bin_file_check() {
   print_output "$(indent "$FILE_BIN_OUT")"
   echo
   print_output "$(indent "$ENTROPY")"
+  HTML_REPORT=1
 
   # probably we can do a lot more stuff in the future ...
+  echo -e "\\n[*] HTML_REPORT:$HTML_REPORT" >> "$LOG_FILE"
```

Comment:
Wouldn't it be possible to add this line in `module_end_log`? In the long run, we want to remove also `export HTML_REPORT` and `HTML_REPORT=1` from the modules.

Reply:
As it is like the statistics output we should not split these into two. Probably we could think if we really need the "export HTML_REPORT" because of the new logging to the file and I don't think the growing report feature is really needed and makes more troubles as it helps.